src/main/java/com/github/princesslana/eriscasper/data/resource/AuditLogChange.java

```diff
@@ -0,0 +1,20 @@
+package com.github.princesslana.eriscasper.data.resource;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.github.princesslana.eriscasper.data.util.AuditLogChangeDeserializer;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+@SuppressWarnings("unused")
+@Value.Immutable
+@JsonDeserialize(using = AuditLogChangeDeserializer.class)
+public interface AuditLogChange {
+
+  boolean isArray();
```

Comment:
Why do we need this isArray method?

Reply:
eh I guess we don't, it had a use at first never thought to remove it